modules/ROOT/pages/afm-view-schedules-private-location.adoc

```diff
@@ -5,18 +5,20 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can find out how many schedules a private location is used by. When a private location is used by a schedule, a worker to run the test associated with the schedule is created in that location. Therefore, seeing the number of schedules that use a private location shows you how many workers are running in that location.
+You can determine the number of schedules that use a private location. 
```

Comment:
Ah, at first I thought that "determine" was the ability to configure rather than view....

Reply:
How about...:
```suggestion
View the number of schedules that use a private location by navigating to that location.
```
??? Isn't this kind of gratuitous?

Reply:
Yeah...this topic could be one sentence somewhere else. Something to look at when you really get into the weeds with this legacy stuff.